hooks/useYouTubeConnection.ts

```diff
@@ -0,0 +1,38 @@
+import { useEffect, useState } from "react";
+
+const useYouTubeConnection = (
+  artistAccountId: string | undefined,
+  userId: string | undefined
+) => {
+  const [channelName, setChannelName] = useState<string | null>(null);
+
+  useEffect(() => {
+    if (!artistAccountId || !userId) {
+      setChannelName(null);
+      return;
+    }
+
+    const fetchChannelInfo = async () => {
+      const params = new URLSearchParams({
+        artistAccountId,
+        userId,
+        include_branding: "true",
```

Comment:
If you have this hard-coded to `true` what's the purpose of including this param at all?

principle: YAGNI
- learn more: https://es.wikipedia.org/wiki/YAGNI